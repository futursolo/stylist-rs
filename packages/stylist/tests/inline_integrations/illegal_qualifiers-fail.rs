fn main() {
    let _ = stylist::css! {
        (not a function) 42 117.9f32 / ^ @ .okay ::also(okay) {
            border: black;
        }
    };
}
