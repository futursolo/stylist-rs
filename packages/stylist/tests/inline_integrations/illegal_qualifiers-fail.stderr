error: expected a valid part of a scope qualifier, not a block
 --> $DIR/illegal_qualifiers-fail.rs:3:9
  |
3 |         (not a function) 42 117.9f32 / ^ @ .okay ::also(okay) {
  |         ^^^^^^^^^^^^^^^^

error: only string literals are allowed in selectors
 --> $DIR/illegal_qualifiers-fail.rs:3:26
  |
3 |         (not a function) 42 117.9f32 / ^ @ .okay ::also(okay) {
  |                          ^^

error: only string literals are allowed in selectors
 --> $DIR/illegal_qualifiers-fail.rs:3:29
  |
3 |         (not a function) 42 117.9f32 / ^ @ .okay ::also(okay) {
  |                             ^^^^^^^^

error: unexpected punctuation in selector
 --> $DIR/illegal_qualifiers-fail.rs:3:38
  |
3 |         (not a function) 42 117.9f32 / ^ @ .okay ::also(okay) {
  |                                      ^

error: unexpected punctuation in selector
 --> $DIR/illegal_qualifiers-fail.rs:3:42
  |
3 |         (not a function) 42 117.9f32 / ^ @ .okay ::also(okay) {
  |                                          ^
